namelists/oifs/43r3/awicm3/202011/fort.4

```diff
@@ -0,0 +1,511 @@
+! -----------------------------------------------------------------------------
+! *** NAMECECFG: EC-Earth specific configuration parameters
+! -----------------------------------------------------------------------------
+&NAMECECFG ! DONE
+    ECE_CPL_NEMO_LIM = false,        ! Whether IFS is coupled to NEMO/LIM
+    ECE_CPL_FESOM_FESIM = true,  ! Whether IFS is coupled to FESOM/FESIM
+/
+&NAMAWICFG
+    ECE_AWI_CPL_FESOM = true, ! Whether IFS is coupled to FESOM2
+/
+
+&NAMFOCICFG
+/
+
+&NAMIOS
+  CFRCF            = "./rcf",
+  CIOSPRF          = "./srf",
+/
+&NAMFPG
+  NFPLEV           = 60,
+  NFPMAX           = 159,
+/
+&NAMCVER
+  LVERTFE          = TRUE,
+  NVSCH            = 3,
+  LRNHC1           = FALSE,
+  LVFE_GWMPA       = TRUE,
+  LVFE_GW          = TRUE,
+  LVFE_Z_TERM      = TRUE,
+  LVFE_X_TERM      = TRUE,
+  LVFE_LAPL_BC     = TRUE,
+  LVFE_LAPL        = TRUE,
+/
+&NAMMCC
+  LMCCIEC          = FALSE,
+  LMCCEC           = TRUE,
+  LMCC04 	   = FALSE,
+  LMCCDYNSEAICE    = TRUE,
+/
+&NAMTRANS
+  LFFTW		   = FALSE,
+/
+&NAMRES
+  NFRRES           = 1,
+  NRESTS           = -1,-48,
+/
+&NAERAD
+  NRPROMA        = -8,
+  LCMIP6         = TRUE,
+  SSPNAME        = 'historical',
+  CMIP6DATADIR   = '/work/ollie/jstreffi/input/oifs-43r3/TL159L60//../cmip6-data/',
+  LA4xCO2        = FALSE,
+  L1PCTCO2       = FALSE,
+  LCMIP5         = FALSE,
+  CMIP5DATADIR   = '/work/ollie/jstreffi/input/oifs-43r3/TL159L60//../cmip5-data/',
+  NCMIPFIXYR     = 1990,
+  NRCP           = 0,
+  LHVOLCA        = FALSE,
+  CRTABLEDIR     ='/work/ollie/jstreffi/runtime/awicm-3.1/DRH6/work//rtables/',
```

Comment:
perhaps the default can point to a pool dir already. I know it's changed later, but its a bit strange to have it point to an old exp in this file.